src/helpers.js

```diff
@@ -154,9 +153,25 @@ export function normalizeSwagger(parsedSpec) {
               if (!operation[inhName]) {
                 operation[inhName] = inherits[inhName]
               }
+              else if (inhName === "parameters") {
+                var exists = false
+                for (const item in inherits[inhName]) {
```

Comment:
`inherits[inhName]` is an array. Should use `for... of`. Following code would also be shorter as no need to deref into array index.